experimental/CliffordAlgOrd/src/IO.jl

```diff
@@ -0,0 +1,73 @@
+##################################################
+#
+#  Clifford algebra and elements
+#
+##################################################
+ 
+##### Algebra #####
+function Base.show(io::IO, ::MIME"text/plain", C::CliffordAlgebra)
+  io = pretty(io)
+  print(io, "Clifford algebra of quadratic space with Gram matrix\n")
+  print(io, Indent())
+  show(io, "text/plain", gram_matrix(C))
+  print(io, Dedent(), "\ndefined over $(base_ring(C))")
```

Comment:
```suggestion
  print(io, Dedent(), "\ndefined over ", Lowercase(), base_ring(C))
```
the base ring needs to get lowercased